Ports/jakt/package.sh

```diff
@@ -10,6 +10,11 @@ archive_hash='d7b5e772074a44f428facd230ac9dafdcf2034c188e31715f592b6058162e4d7'
 files=(
     "https://github.com/SerenityOS/jakt/archive/${commit_hash}.tar.gz#${archive_hash}"
 )
+launcher_name='Jakt'
+launcher_category='D&evelopment'
+launcher_command='/usr/local/bin/jakt --repl'
+launcher_run_in_terminal='true'
+icon_file='https://raw.githubusercontent.com/djwisdom/digital-assets/95c35a3e3391fb178ad2292856bcf3d2090f404d/ports/jakt/app-jakt.png'
```

Comment:
This feels way too detailed to be an icon.

Also, we should discuss this with the Jakt people (@alimpfard :yakkie:) and possibly check in the logo into their repository.

Reply:
On second thought, I can do without the logo since I also don't know how Jakt should be presented in a logo. I can omit the icon_file line and resubmit.